### PR TITLE
Resolves #599 Adding new script MEMOS Method B WCOM

### LINKED ISSUE #599: Method B budget breakdown on SPEC/WCOM
I know back in the day (Anoka circa 2014) we talked about this idea at one point.  Basically would breakdown the deductions and difference between the spenddown amount and the recipient amount.  Didn't see this issue out there, so thought I'd throw it out there.

### PR DESCRIPTION
**Pull request by IlseFerris**
Blip: New MEMOS script creates detailed WCOM regarding spenddown vs. recipient amount for method B HC cases.

### CODE REVIEW

**Comment by theVKC** [comment]
Is this tested, @IlseFerris? If not, I might delay until Monday in order to give it enough time for a good test.

**Comment by IlseFerris** [comment]
Yes, tested and ready to roll

**Comment by theVKC** [comment]
Can you add the button on the main menu to this?

**Comment by theVKC** [comment]
@IlseFerris: are you running into issues adding the button? Want my help?

**Comment by IlseFerris** [comment]
I got a merge conflict when I tried to pull the two together. 
I'm just busy right now, so it's your call.

**Comment by theVKC** [comment]
Don't pull the two together. That would be disastrous.

1. Copy your menu update
2. Switch to the good branch (Method-B)
3. Paste the menu update
4. Commit again
5. Sync

At that point, let me know if you have a merge conflict.

**Comment by IlseFerris** [comment]
Done and done

**Comment by theVKC** [comment]
Well, that caused the merge conflict. I'll investigate next week, possibly at hackathon.

**Comment by theVKC** [comment]
Actually...

Are you pulling the main menu from GitHub (on the web) before adding the new thing? Or are you working within GitHub desktop and Atom?

I think the conflict is about your line endings. It makes me think you're copy/pasting from GitHub on the web (and if you aren't copying the raw file, this causes issues, which is why I recommend desktop).

Can you check that for me? :)

**Comment by IlseFerris** [comment]
I was working from GitHub desktop and Atom. I did NOT copy and paste into the GitHub on the web. I got a merge conflict on desktop initially too, which is why I pulled it in initially in its own branch. Ugh :(

File: Script Files/MEMOS/MEMOS - METHOD B WCOM.vbs [hunk 1]
```diff
@@ -0,0 +1,241 @@
+'STATS GATHERING----------------------------------------------------------------------------------------------------
+ name_of_script = "MEMOS - METHOD B WCOM.vbs"
+ start_time = timer
+
+ 'LOADING FUNCTIONS LIBRARY FROM GITHUB REPOSITORY===========================================================================
```
Comment (theVKC) [inline Script Files/MEMOS/MEMOS - METHOD B WCOM.vbs]: Wrong FuncLib, @IlseFerris...

File: Script Files/MEMOS/MEMOS - METHOD B WCOM.vbs [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+income = Ltrim(Income)
+SD = Ltrim(spenddown)
+medi_part_a = Ltrim(medi_part_a)
+medi_part_b = Ltrim(medi_part_b)
+
+'Shows the dialog
+Do
+  err_msg = ""
+  Do
+		Dialog MEMOS_LTC_METHOD_B_dialog
```
Comment (theVKC) [inline Script Files/MEMOS/MEMOS - METHOD B WCOM.vbs]: Why's this tabbed over so?

File: Script Files/MEMOS/MEMOS - MAIN MENU.vbs [hunk 3]
```diff
@@ -44,39 +44,40 @@ IF IsEmpty(FuncLib_URL) = TRUE THEN	'Shouldn't load FuncLib if it already loaded
...
   Text 65, 180, 355, 20, "--- Sends a MEMO informing client that they need to report information regarding the birth of their child, and/or pregnancy end date, within 10 days or their case may close."
-  Text 95, 205, 345, 20, "--- Sends a WCOM informing the client of postponed verifications that MAXIS won't add to notice correctly by itself."
-  Text 75, 230, 340, 10, "--- Sends a MEMO when a client reports significant change requiring additional action."
-  Text 80, 245, 300, 10, "--- Sends a SPEC/LETR informing client that they have an Employment and Training appointment."
+  Text 90, 205, 355, 10, "--- Sends a WCOM informing the client of postponed verifs that MAXIS won't add to notice correctly by itself."
+  Text 75, 220, 340, 10, "--- Sends a MEMO when a client reports significant change requiring additional action."
+  Text 80, 235, 300, 10, "--- Sends a SPEC/LETR informing client that they have an Employment and Training appointment."
+  Text 5, 5, 235, 10, "Memos scripts main menu: select the script to run from the choices below."
+  Text 70, 120, 350, 10, "---NEW 01/2016 Makes detailed WCOM regarding spenddown vs. recipient amount for method B HC cases"
```
Comment (theVKC) [inline Script Files/MEMOS/MEMOS - MAIN MENU.vbs]: Should be new for 02/2016 as this isn't included in the v2016.01.2 release...

**Comment by theVKC** [comment]
This is a head scratcher. Please make the changes in the comments and we'll check it tomorrow at hackathon. :)

**Comment by theVKC** [comment]
@IlseFerris: can we update the FuncLib block (and other items in the comments)?

**Pull request merged**